.github/labeler.yml

```diff
@@ -0,0 +1,38 @@
+lib:
```

Comment:
This label doesn't exist in GitHub yet, see also related discussion about labels at, https://trac.osgeo.org/grass/wiki/GitMigration#Migrationoftracissues

Reply:
`lib` or `libraries`? I'd prefer the latter for being less cryptic.

Reply:
I chose `lib` as it's shorter, but `libraries` is good for me too.